influxdb/server.sls

```diff
@@ -25,8 +25,82 @@ influxdb_service:
   service.running:
   - enable: true
   - name: {{ server.service }}
+  # This delay is needed before being able to send data to server to create
+  # users and databases.
+  - init_delay: 5
```

Comment:
eventually we could add another state to wait for the InfluxDB HTTP service to be up and ready. But I agree that a hardcoded delay is enough for now.